### ISSUE TITLE
Add support for multiple networks 

### ISSUE THREAD

**Issue by tarasherasymchuk** [utu-protocol/market#9]
Support for scanning multiple EVM-compatible networks should be added to the defi-scanner starting with Polygon and then Gnosis chains.
This shall include both DeFi Protocols and Ocean assets/pools/datatokens.

**Comment by databu** [comment]
Also, support for Polygon and Gnosis chains should each include both test nets (Polygon: Mumbai, Gnosis: ?) and main nets.

**Comment by tarasherasymchuk** [comment]
@databu How should we change TrustEntity or TrustRelationship so we can include an identifier of a network that was scanned?

**Comment by databu** [comment]
Yes, though an entity might be active on multiple networks. So probably it would be best to have an array of chain ids. I think you could just add this to the generic `properties` object, which both Entity and Relationship types have.
